common_issues.md

```diff
@@ -43,24 +43,23 @@ Check your INT wire, there is either a bad connection or you have it connected t
 
 ## The trackers are connected to my wifi but don't turn up on SlimeVR
 
-This is most likely a firewall issue, go to SlimeVR Server folder and run `firewall.bat` as administrator.
+Check that you do not have two copies of the SlimeVR server running, as only one of them will show trackers connected.
 
-If you are still having trouble, try manually adding the SlimeVR Server to your firewall. 
-
-Go to **Settings** > **Network & Internet** then click on the text link **Window Firewall** (you may have to scroll down). 
-
-In the firewall window, click the link **Allow An app through firewall**.
-If your options are greyed out on this page, click the **Change Settings** button. 
+If only one server is running, this is most likely a firewall issue, go to SlimeVR Server folder and run `firewall.bat` as administrator.
 
-Click the **Allow another app...** button, and then **Browse...** on the window that pops up. 
-
-In the text box next to the words File Name type `*.*` and press enter before navigating to `slimevr.jar` in your SlimeVR server folder and select it (if you cannot see files in this folder try typing `*.*` and pressing enter again to show all files). 
+If you are still having trouble, try manually adding the SlimeVR Server to your firewall. 
 
-Finally, make sure both public and private check boxes are selected here.
+1. Go to **Settings** > **Network & Internet** then click on the text link **Window Firewall** (you may have to scroll down). 
+1. In the firewall window, click the link **Allow an app through firewall**.
+1. Click the **Allow another app...** button, and then **Browse...** in the opened **Add an app** window. 
+	1. If your options are greyed out in the **Allowed apps** window, click the **Change Settings** button to allow changes.
```

Comment:
Since this is the only action item, it can be either converted to bullet list item (`*` or `-`) so it stands out or put inside previous list item as a second sentence.

Reply:
Done